src/Compilers/Core/Portable/Syntax/SyntaxNode.cs

```diff
@@ -981,6 +981,7 @@ public SyntaxNode FindNode(TextSpan span, bool findInsideTrivia = false, bool ge
                     if (parent == null || parent.FullWidth != node.FullWidth) break;
                     // prefer child over compilation unit
                     if (parent == cuRoot) break;
+
                     node = parent;
```

Comment:
>node = parent; [](http://example.com/codeflow?start=20&length=14)

It doesn't look like there are any meaningful changes in this file. Please revert it to its original state.